http_service/bugbug_http/models.py

```diff
@@ -32,41 +30,34 @@
     "testlabelselect",
     "testgroupselect",
 ]
+
 DEFAULT_EXPIRATION_TTL = 7 * 24 * 3600  # A week
+redis = Redis.from_url(os.environ.get("REDIS_URL", "redis://localhost/0"))
 
 
-MODEL_LAST_LOADED: Dict[str, datetime] = {}
-MODEL_CACHE: Dict[str, Model] = {}
+def load_model_for_service(model_name):
```

Comment:
Actually, we can just get rid of this function completely, and add the try-except around https://github.com/mozilla/bugbug/blob/e00e951659e32d8a509801494f5b878529b2b52b/http_service/bugbug_http/check_models.py#L19

Reply:
Don't forget to update the `get_model` calls in download_models.py too (it used to be in check_models.py, it is now in download_models.py).

Reply:
I hadn't noticed you did modify check_models.py! So, you just need to apply that change to download_models.py instead ;)